components/divider/divider.component.ts

```diff
@@ -17,9 +17,11 @@ import { InputBoolean } from 'ng-zorro-antd/core/util';
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <span *ngIf="nzText" class="ant-divider-inner-text">
-      <ng-container *nzStringTemplateOutlet="nzText">{{ nzText }}</ng-container>
-    </span>
+    @if (nzText) {
```

Comment:
Let's remove the `NgIf` import :)

Reply:
True, forgot it. Will do it today ! Thanks for the review